tests/components/fritz/snapshots/test_image.ambr

```diff
@@ -1,10 +1,10 @@
+  b"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00t\x00\x00\x00t\x01\x00\x00\x00\x00|\xe1+\x8a\x00\x00\x00\xe0IDATx\xda\x9dU\xc1\x11\xc30\x0c\xe2\xb2\x00\xfbo\xc9\x06*\xc8\xee\xa3\xaf^\xf0\xc7\xce\x03\t\x84\xa4\x00\x00G\xa04~\xe2\xc1\xef\xf9\xf7m\xf0PF\x83\xf4\xb3\xc0\x07\xb8\xd8\xf1\xb3\xc5\x87D\x8f\x87\x93\x8f\xd8\xe1\x9d\xda@g\x9fJ\x7fB\xdcS\xd5?\xd4\xad\xdf\xee\xa5\x84*\xf4\x1b\x88t\x80\xfd\xe7{<\xd6{\x07P\x14\xbc\xc7\x1f\xdb\xb4DT\xf1\x8f\xf6\x841\x015\xf5\x97\x9dK\xf1w\n\n\xff\xa1m@\xf9.\xf4\x7f\x85\x9bC44\xfd\xe7\xfcq0\x01\xaa\xfc;\x00\xbe\xa4\xa6~J\xdd\xee\x18\xa8\xf1/\xc4\xb5[\x88\x8d\xff:\xca\xe3\x01\x9b\xfd\x93\xe9\x89\xfdD\xd5\xff\x99\xbf\xbb?ba5\xffv\xff,\xd0\xce\xff]\x7fn\xff4A\xb9\xff\xa8\xd5_\xe29\xc7=\xa8\xdc\xbf\xf7'\x04t\xfa\x95\xee\xdfI\xec\xea\xef\x00\x8a\x93\x85\xfe\x0f\x80/\xb1\xfdI7\xe3s\x00\x00\x00\x00IEND\xaeB`\x82"
 # ---
 # name: test_image_update[fc_data0]
-  b'\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x94\x00\x00\x00\x94\x01\x00\x00\x00\x00]G=y\x00\x00\x00\xf9IDATx\xda\xedV\xc1\r\xc40\x0cB\xb7\x80\xf7\xdf\x92\r\\\xb0\xfb\xeb\xe7\xaa\xf0l\xd4\xaaQ\x1e\xc8\x06L\x8a~,\xe2;{s\x06\xa0\xd8z9\xdb\xe6\x0f\xcf\xf5\xef\x99\xf0J\x0f\x85\x86*o\xcf\xf1\x04\x04\x1ak\xb6\x11<\x97\xa6\xa6\x83x&\xb32x\x86\xa4\xab\xeb\x08\x7f\x16\xf5^\x11}\xbd$\xb0\x80k=t\xcc\x9f\xfdg\xfa\xda\xe5\x1d\xe3\t\x8br_\xdb3\x85D}\x063u\x00\x03\xfd\xb6<\xe2\xeaL\xa2y<\xae\xcf\xe3!\x895\xbfL\xf07\x0eT]n7\xc3_{0\xd4\xefx:\xc0\x1f\xc6}\x9e\xb7\x84\x1e\xfb\x91\x0e\x12\x84\t=z\xd2t\x07\x8e\x1d\xc9\x03\xc7\xa9G\xb7\x12\xf3&0\x176\x19\x98\xc8g\x8b;\x88@\xc6\x7f\x93\xa9\xfbVD\xdf\x193\xde9\x1d\xd1\xc3\x9ev`E\xf2oo\xa3\xe1/\x847\xad\x8a?0t\xffN\xb4p\xf35\xf3\x7f\x80\xad\xafS\xf7\x1bD`D\x8f\xef\x9f\xf0\xe0\xec\x02\xa4\xc0\x83\x92\xcf\xf3\xf9a\x00\x00\x00\x00IEND\xaeB`\x82'
+  b"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00t\x00\x00\x00t\x01\x00\x00\x00\x00|\xe1+\x8a\x00\x00\x00\xe0IDATx\xda\xa5U\xc1\r\x031\x0cB]\xc0\xfbo\xc9\x06\x14\xec\xfb\xf4U\x95FJ\x94{\x10c\xb0}\x000\x94\xb7\x0f\xaf\x17>\xd7\xb7o\x8c44\x1a3\xbe\x16x\x03\xc1`\xe5k\x87Oh'\xf1\x07\xde\xd1\xcd\xa1\xc2\x877\x13]U\xfey\xe2Y\x95\xfe\xd2\x1a\xe0\xd0\x9bD\x91\x7f\xfcO\xfa\xca\xedg\xbc\xb1\xb4\xfb\x8a\x87\x16\xa2\x88\x1f\xf0\x11a\xc1_6/\xd1#\xc2\xb0\xf0/\xac}\xba\xfe\xd9\xe4\xaf\xd8n\xf1B\xbf\xcb_)<\xf3\xcfn\xf2\xc7\xba\x9f\xfam\xf4{\x1eQ\x82\xb3\xd1O;=\xae\x80\xc9\xaa\x7f2>\xf2\xd04\xf5k\xf0\xc4\xfe\xcc\x80f\xfeD\xfc}\x01\xe8\xfc\xdf\xc1u{*\xfd\xd3\xbe7@\xa7\xd4/5\x94\x06\xae\xfa\xff\xa6\xe7\xe6_\xe2\x97\xba\x99\x80\xe5\xfcO\xeby\x03l\xff?\xb8\xf8l\xe7\xaf\xa1j\xf4{\x03\x17\xfa\xb4\x19\xc7\xc5\xe1\xd3\x00\x00\x00\x00IEND\xaeB`\x82"
```

Comment:
the changes to the qr code images (_svg files_) are based on the new `border=0` property which just improves the _readability_ of the qr codes by removing a surrounding white frame from the qr codes - also this I've duple checked and it still works :+1: